src/main/java/httpServer/App.java

```diff
@@ -1,7 +1,16 @@
 package httpServer;
 
+import java.net.ServerSocket;
+
 public class App {
-    public String hello() {
-        return "Hello world";
+    public static void main(String[] args) {
+        try {
+            var serverSocket = new ServerSocket(5000);
+            var executor = new CurrentThreadExecutor();
```

Comment:
Fine for now - but think you should swap this out for a proper threaded executor quite soon. The `CurrentThreadExecutor` should really just be used for tests.

Reply:
I'll swap this. Thanks, Matt